### ISSUE TITLE
Would this work on aarch64?

### ISSUE THREAD

**Issue by syed-ahmed** [facebook/thpp#17]
I was wondering if the install would work on a linux aarch64 machine. Has anybody tried it?

**Comment by ebetica** [comment]
Yes, I have this built and installed on 64-bit Manjaro, which afaik is based off of Arch.

**Comment by syed-ahmed** [comment]
Hi @ebetica, I was referring to the arm architecture called aarch64. One such machine is a Jetson TX1. Any chance you might have installed on a similar machine?
